### ISSUE TITLE
How to get the accuracy on ISBI dataset

### ISSUE THREAD

**Issue by shawnricecake** [hanyoseob/pytorch-UNET#2]
Hi, 

I found that you have shown the loss in the training and testing part,

But how to compute the accuracy like the paper you cited in the readme?

Thanks!
